iwyu_ast_util.cc

```diff
@@ -1713,6 +1715,33 @@ bool IsDerivedToBasePtrConvertible(const Type* derived_ptr_type,
   return false;
 }
 
+bool IsBaseToDerivedMemPtrConvertible(
```

Comment:
Could you add a small example of two types/functions being base-to-derived member pointer convertible in the doc comment? Just something to trigger the brain what this looks like in actual code.